var/spack/repos/builtin/packages/py-logilab-common/package.py

```diff
@@ -11,6 +11,7 @@ class PyLogilabCommon(PythonPackage):
     homepage = "https://www.logilab.org/project/logilab-common"
     url      = "https://pypi.io/packages/source/l/logilab-common/logilab-common-1.2.0.tar.gz"
 
+    version('1.4.2', sha256='cdda9ed0deca7c68f87f7a404ad742e47aaa1ca5956d12988236a5ec3bda13a0')
     version('1.2.0', sha256='d4e5cec3be3a89f06ff05e359a221e69bd1da33cb7096cad648ddcccea8465b7')
 
     extends('python', ignore=r'bin/pytest')
```

Comment:
Setuptools is probably required for both build/run. Also, six should be `six@1.4.0:`. There are a few additional dependencies you can add if you want:
```python
install_requires = [                                                            
    'setuptools',                                                               
    'six >= 1.4.0',                                                             
]                                                                               
tests_require = [                                                               
    'pytz',                                                                     
    'egenix-mx-base',                                                           
]                                                                               
                                                                                
if sys.version_info < (2, 7):                                                   
    install_requires.append('unittest2 >= 0.5.1')                               
if os.name == 'nt':                                                             
    install_requires.append('colorama')
```
The `unittest2` dependency is the only one you definitely should add.

Reply:
Done.